core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -32,7 +32,8 @@ class CityInfo {
     var tiles = HashSet<Vector2>()
     var workedTiles = HashSet<Vector2>()
     var isBeingRazed = false
-
+    var attacksThisTurn = 0
+    val range = 2
```

Comment:
Since the range is always 2, I don't think this should be a per city parameter
If you're using it for convenience because it's used in several places, make it transient, so it isn't serialised in the saved game